streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -168,7 +168,7 @@
     public <KR> KStream<KR, V> selectKey(final KeyValueMapper<? super K, ? super V, ? extends KR> mapper) {
         Objects.requireNonNull(mapper, "mapper can't be null");
 
-        final ProcessorGraphNode<K, V> selectKeyProcessorNode = internalSelectKey(mapper);
+        final ProcessorGraphNode<K, V> selectKeyProcessorNode = internalSelectKey(mapper, NamedInternal.empty());
```

Comment:
What is this change about?

Reply:
The method `internalSelectKey` now accept an `NamedInternal` instance. As this method is private we are expecting to not pass a null argument. Passing an empty `NamedInternal.empty()` allow us to not check for nullable witihin the method `internalSelectKey`